features/tracker-allowlist.json

```diff
@@ -1117,6 +1117,14 @@
                     }
                 ]
             },
+                    },
```

Comment:
Uh the formatting sems weird here. Did something go awry during a rebase/merge?

There should be an existing entry for `greylabeldelivery.com`, so you can add there.

FYI, the "reason" field should link to a GitHub issue (descriptive comments are deprecated).

Reply:
What Ben said, see https://github.com/duckduckgo/privacy-configuration/pull/949 for an example of a content blocking mitigation.